api/task_status.go

```diff
@@ -24,13 +26,15 @@ type TaskStatus struct {
 // taskStatusHandler handles the task status endpoint
 type taskStatusHandler struct {
 	store   *event.Store
+	drivers map[string]driver.Driver
```

Comment:
Keeping track of where this map is used -- now a third obj sharing view of the same map.

Reply:
:nod: and there's fourth object (overall status API) down the line. I may go ahead and refactor this after Beta (thinking about the code freeze) unless you think it should be done beforehand.